src/reanimated2/layoutReanimation/web/componentUtils.ts

```diff
@@ -17,15 +17,12 @@ import { _updatePropsJS } from '../../js-reanimated';
 import type { ReanimatedHTMLElement } from '../../js-reanimated';
 import { ReduceMotion } from '../../commonTypes';
 import type { StyleProps } from '../../commonTypes';
-import { useReducedMotion } from '../../hook/useReducedMotion';
+import { isReducedMotion } from '../../PlatformChecker';
 
 function getEasingFromConfig(config: CustomConfig): string {
   const easingName = (
-    config.easingV !== undefined &&
-    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-    config.easingV!.name in WebEasings
-      ? // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        config.easingV!.name
+    config.easingV !== undefined && config.easingV?.name in WebEasings
```

Comment:
what about simply `config?.easingV?.name in WebEasings`

Reply:
Typescript for some reason doesn't allow to pass `undefined` to `in` operator but it's not something simple `as` can't fix xD